tsconfig.json

```diff
@@ -0,0 +1,34 @@
+{
+  "compilerOptions": {
+    "baseUrl": ".",
+    "rootDir": "./",
+    "outDir": "dist/tsc/",
+    "noEmitOnError": true,
+    "sourceMap": true,
+    "module": "amd",
+    "target": "es5",
+    "types": [
+      "mendix-client",
+      "jasmine",
+      "core-js"
+    ],
+    "moduleResolution": "node",
+    "declaration": false,
+    "removeComments": true,
+    "noLib": false,
+    "watch": false,
+    "forceConsistentCasingInFileNames": true,
+    "noFallthroughCasesInSwitch": true,
+    "noImplicitAny": true,
```

Comment:
Option `strict`  Enable all strict type checking options. 
Enabling --strict enables --noImplicitAny, --noImplicitThis, --alwaysStrict and --strictNullChecks.